src/main/java/org/folio/dew/batch/bulkedit/jobs/updatejob/BulkEditUpdateItemRecordsJobConfig.java

```diff
@@ -0,0 +1,76 @@
+package org.folio.dew.batch.bulkedit.jobs.updatejob;
+
+import org.folio.dew.batch.JobCompletionNotificationListener;
+import org.folio.dew.batch.bulkedit.jobs.JobConfigReaderHelper;
+import org.folio.dew.domain.dto.Item;
+import org.folio.dew.domain.dto.ItemFormat;
+import org.springframework.batch.core.ItemWriteListener;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.core.launch.support.RunIdIncrementer;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.batch.item.ItemReader;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.batch.item.file.FlatFileItemReader;
+import org.springframework.batch.item.file.LineMapper;
+import org.springframework.batch.item.file.builder.FlatFileItemReaderBuilder;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.FileSystemResource;
+
+import static org.folio.dew.domain.dto.EntityType.ITEM;
+import static org.folio.dew.domain.dto.ExportType.BULK_EDIT_UPDATE;
+import static org.folio.dew.utils.Constants.FILE_NAME;
+
+@Configuration
+public class BulkEditUpdateItemRecordsJobConfig {
+
+  @Bean
+  public Job bulkEditUpdateItemRecordsJob(
+    Step bulkEditUpdateItemRecordsStep,
+    JobBuilderFactory jobBuilderFactory,
+    JobCompletionNotificationListener completionListener) {
+    return jobBuilderFactory
+      .get(BULK_EDIT_UPDATE.getValue() + "-" + ITEM.getValue())
```

Comment:
the same - https://github.com/folio-org/mod-data-export-worker/pull/176/files#r841413383

Reply:
fixed